services/asset/properties/schemas.go

```diff
@@ -8,81 +8,11 @@ import (
 	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
 )
 
-var validPropertyTypes = []string{"NUMERIC", "ENUM", "TEXT", "TIMESTAMP", "DATE", "TIME", "BOOLEAN", "GEOPOINT"}
+var validPropertyTypes = []string{"NUMERIC", "ENUM", "TEXT", "TIMESTAMP", "DATE", "TIME", "BOOLEAN", "GEOPOINT", "TLE"}
+var validNodeTypes = []string{"ASSET", "GROUP", "COMPONENT"}
+var validNodeKinds = []string{"GENERIC", "SATELLITE", "GROUND_STATION"}
 
-var propertyFieldSchema = map[string]*schema.Schema{
-	"id": {
-		Type:     schema.TypeString,
-		Computed: true,
-	},
-	"name": {
-		Type:     schema.TypeString,
-		Required: true,
-	},
-	"description": {
-		Type:     schema.TypeString,
-		Optional: true,
-	},
-	"created_at": {
-		Type:        schema.TypeString,
-		Computed:    true,
-		Description: "When it was created",
-	},
-	"created_by": {
-		Type:        schema.TypeString,
-		Computed:    true,
-		Description: "Who created it",
-	},
-	"last_modified_at": {
-		Type:        schema.TypeString,
-		Computed:    true,
-		Description: "When it was last modified",
-	},
-	"last_modified_by": {
-		Type:        schema.TypeString,
-		Computed:    true,
-		Description: "Who modified it the last",
-	},
-	"value": {
-		Type:     schema.TypeString,
-		Optional: true,
-	},
-	"type": {
-		Type:         schema.TypeString,
-		Required:     true,
-		ValidateFunc: validation.StringInSlice(validPropertyTypes, false),
-		Description:  helper.AllowedValuesToDescription(validPropertyTypes),
-	},
-}
-
-var geoPointFieldsSchema = map[string]*schema.Schema{
-	"elevation": {
-		Type:     schema.TypeList,
-		MaxItems: 1,
-		Required: true,
-		Elem: &schema.Resource{
-			Schema: propertyFieldSchema,
-		},
-	},
-	"latitude": {
-		Type:     schema.TypeList,
-		MaxItems: 1,
-		Required: true,
-		Elem: &schema.Resource{
-			Schema: propertyFieldSchema,
-		},
-	},
-	"longitude": {
-		Type:     schema.TypeList,
-		MaxItems: 1,
-		Required: true,
-		Elem: &schema.Resource{
-			Schema: propertyFieldSchema,
-		},
-	},
-}
-
-var propertySchema = map[string]*schema.Schema{
+var PropertySchema = map[string]*schema.Schema{
```

Comment:
I still don't like the uppercase
If there's conflicts could you rename it?

Reply:
No it is not a conflict problem. It's about the visibility of this schema outside the properties package. 

The propertySchema is used in terraform-provider-leanspace\services\asset\nodes\schemas.go :

	"properties": {
		Type:        schema.TypeSet,
		Optional:    true,
		Description: "Node properties",
		Elem: &schema.Resource{
			Schema: properties.PropertySchema,
		},
	},
The only way I found to import this schema in nodes package is by using a capital letter.

Reply:
ah yes my bad I misunderstood

Reply:
Oh wait, I have removed the properties from the Node schema. So I can remove the capital letter :)